mri/test/test_optimizer.py

```diff
@@ -48,8 +48,8 @@ def test_reconstruction_fista_fft2(self):
         print("Process test FFT FISTA::")
         for image in self.images:
             fourier = FFT(samples=convert_mask_to_locations(
-                                            fftshift(self.mask)),
```

Comment:
I don't know what these whitelines changes are.
Ideally we should avoid those (even if can be hidden in github review) because they clutter unnecessarily the git history.
When having to write on multiple lines, the best way to do it to avoid having to make whiteline changes is:
```python
var = long_function_call(
	var1=var1,
	var2=var2,
	var3=var3,
)
```
This will limit at best the number of unnecessary changes (whitelines and commas).

Reply:
Yes, but I will not change them here as this file will change fully in future. (We will use reconstructors)